bin/stats

```diff
@@ -161,6 +161,7 @@ if __name__ == "__main__":
     from trellodog import Trellodog
     from optparse import OptionParser
     parser = OptionParser(version=VERSION, usage=usage)
+    parser.add_option("-d", "--days", action="store", type="int", dest="days", help="number of days of activity to return")
```

Comment:
Default is still set on the activity method in the `trellodog.py` module. [This](https://github.com/mannytoledo/trellodog/blob/master/Trellodog/trellodog.py#L59) should default it to 3 days.